packages/pds/src/handle/moderation/index.ts

```diff
@@ -0,0 +1,66 @@
+import * as ui8 from 'uint8arrays'
+import AppContext from '../../context'
+import { REASONOTHER } from '../../lexicon/types/com/atproto/moderation/defs'
+import { UnacceptableHandleValidator } from './validator'
+
+// regexes taken from: https://github.com/Blank-Cheque/Slurs
+/* eslint-disable no-misleading-character-class */
+const explicitSlurRegexes = [
```

Comment:
There's only regexes available for a handful of slurs though, which means this might actually regress some of the previously banned words?

Wouldn't it make sense to import a library that has a more robust list, and then an additional constant or database call to pull out Bluesky custom implementations?